README.md

```diff
@@ -5,65 +5,63 @@
 [![Download Status](https://img.shields.io/npm/dm/node-vault.svg?style=flat-square)](https://www.npmjs.com/package/node-vault)
 [![Dependency Status](https://img.shields.io/david/kr1sp1n/node-vault.svg?style=flat-square)](https://david-dm.org/kr1sp1n/node-vault)
 
-A client for the HTTP API of HashiCorp's [Vault] written for Node.js.
+NodeJS API client for HashiCorp's [Vault]
 
 
-## install
-make sure to use node.js version >= 6
+## Install
+make sure to use node.js version >= 8
 
+    # yarn
+    yarn add node vault
```

Comment:
Should be `yarn add node-vault`.

Reply:
True, oops

Reply:
Fixed https://github.com/DaniGuardiola/node-vault/commit/6627755b641f8d27beef03e89084c696c5a9a670